pmd-java/src/main/resources/category/java/design.xml

```diff
@@ -1136,6 +1136,36 @@ derived from RuntimeException or a checked exception.
 <![CDATA[
 public void foo() throws Exception {
 }
+]]>
+        </example>
+    </rule>
+	
+	<rule name="SignatureDeclareThrowsRuntimeException"
```

Comment:
Maybe the rule should be named `ThrowsClauseDeclaresUncheckedExceptions`, or `AvoidUncheckedExceptionsInSignatures`? IMHO the two verbs glued together (`DeclareThrows`) are very confusing on first read

I think consistency with `SignatureDeclareThrowsException`  can be overlooked, especially since it should be "declares" with an "s" anyway.

@pamidi99 Wdyt?

Reply:
I think `AvoidUncheckedExceptionsInSignatures ` looks good to me. @adangel  Wdyt? Once you both confirm the renaming, will make the change.

Reply:
sounds good :+1: